apps/src/applab/firebaseStorage.js

```diff
@@ -27,8 +27,12 @@ function getRecordsRef(channelId, tableName) {
  * @param {function (string, number)} onError Function to call on error with error msg and http status.
  */
 FirebaseStorage.getKeyValue = function (key, onSuccess, onError) {
-  let keyRef = getKeysRef(Applab.channelId).child(key);
-  keyRef.once("value", object => onSuccess(object.val()), onError);
+  const keyRef = getKeysRef(Applab.channelId).child(key);
+  keyRef.once("value", snapshot => {
+    // Return undefined if the key was not found, otherwise return the decoded value.
+    const value = snapshot.val() === null ? undefined : JSON.parse(snapshot.val());
```

Comment:
Probably the user's program will fail, with the exception message returned by JSON.parse. IIRC It's a quirk of applab that errors in our code are exposed to the Applab user (although I could be wrong about this being the case).